servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -145,17 +173,24 @@ def _get_sys_class_relation_filter_query(self, sys_class_filter):
         return sysparm_query
 
     @staticmethod
-    def _filter_empty_metadata(data):
+    def _apply_field_metadata(data, display_value_list):
```

Comment:
This actually only selects `display_value` or `value` depending on whether it's in `display_value_list`. Can we maybe rename the function to `select_metadata_field`... or something better :smile:

Reply:
yeah, I couldn't think better.